.github/workflows/verify.yml

```diff
@@ -0,0 +1,24 @@
+name: Verify Contribution
+
+on: pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.repo.full_name != github.repository && github.actor != 'dependabot[bot]'
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Get Changed Files
+      id: files
+      uses: jitterbit/get-changed-files@v1
+      with:
+        format: 'json'
+    - run: |
+        readarray -t all_files <<<"$(jq -r '.[]' <<<'${{ steps.files.outputs.all }}')"
+        for file in ${all_files[@]}; do
+          [[ '${file}' == *.json ]] || (echo "Not a JSON file: ${file}" && exit 1)
+          [[ '${file}' == commandGroups/* ]] || [[ '${file}' == profiles/* ]] || (echo "Wrong location: ${file}" && exit 1)
+        done
```

Comment:
This is great. Should we maybe add a label to allow for changes to the typescript?